README.md

```diff
@@ -13,8 +13,8 @@ Ladybug is a Python library to load, analyze and modify EneregyPlus Weather file
 
 This repository includes the core library which is the base for Ladybug. For plugin-specific questions and comments refer to [ladybug-grasshopper](https://github.com/ladybug-tools/ladybug-grasshopper) or [ladybug-dynamo](https://github.com/ladybug-tools/ladybug-dynamo) repositories.
 
-## note
-For the legacy Ladybug Grasshopper plugin see [this repository](https://github.com/mostaphaRoudsari/ladybug).
+## Note
+For the legacy, Ladybug Grasshopper plugin see [this repository](https://github.com/mostaphaRoudsari/ladybug).
```

Comment:
We don't need the comma here since legacy is being used as an adjective. You could capitalize `Legacy` and `Plugin`, though, to make it clear that we are talking about one thing. Also, if you could update the link to the repository to be (https://github.com/ladybug-tools/ladybug-legacy), that would be great.